config/olm/openshift/0.10.0/manifests/dynatrace-monitoring.v0.10.0.clusterserviceversion.yaml

```diff
@@ -764,4 +820,5 @@ spec:
     - dynatrace-monitoring.v0.7.1
     - dynatrace-monitoring.v0.8.0
     - dynatrace-monitoring.v0.8.1
-  version: 0.9.2
+    - dynatrace-monitoring.v0.9.5
```

Comment:
Don't think this line should be here. It stops at 0.8.1 for the last old format CSV.